eventbroadcaster/notifiersink_test.go

```diff
@@ -0,0 +1,502 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2024 Datadog, Inc.
+
+package eventbroadcaster
+
+import (
+	"context"
+	"fmt"
+	"github.com/DataDog/chaos-controller/api/v1beta1"
+	"github.com/DataDog/chaos-controller/eventnotifier"
+	notifTypes "github.com/DataDog/chaos-controller/eventnotifier/types"
+	"github.com/DataDog/chaos-controller/log"
+	"github.com/DataDog/chaos-controller/mocks"
+	"github.com/google/uuid"
```

Comment:
```suggestion

	"github.com/google/uuid"
```
imports aren't formatted